cmd/create/cluster/cmd.go

```diff
@@ -3135,8 +3192,8 @@ func buildCommand(spec ocm.Spec, operatorRolesPrefix string,
 		}
 	}
 	if spec.ClusterAdminUser != "" {
-		command += fmt.Sprintf(" --cluster-admin-user %s", spec.ClusterAdminUser)
-		command += fmt.Sprintf(" --cluster-admin-password %s", spec.ClusterAdminPassword)
+		command += " --create-admin-user"
+		command += fmt.Sprintf(" --create-admin-password %s", spec.ClusterAdminPassword)
```

Comment:
should we hide the password here?

Reply:
I believe we may want to have the password shown since it might interfere with any automation if the user decides to copy and paste the command with a redacted password, as indicated here: https://redhat-internal.slack.com/archives/C04C3G5MDKN/p1695329663534979